pkg/detectors/apiflash/apiflash.go

```diff
@@ -24,7 +24,8 @@ var (
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
 	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"apiflash"}) + `\b([a-z0-9]{32})\b`)
-	urlPat = regexp.MustCompile(detectors.PrefixRegex([]string{"apiflash"}) + `\b([a-zA-Z0-9\S]{21,30})\b`)
```

Comment:
I wasn't sure of the original intention behind detecting a URL specifically for verification, as we can use any fixed URL with an access key for this purpose. To improve flexibility, I removed the URL pattern and introduced a fixed Google URL.

Reply:
that makes sense to me